Simulation/Headers/Components/com_Physics.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <glm/glm.hpp>
+
+#include "ECS/Types.h"
+
+#include "Algorithms/Physics.h"
+
+#include "Components/com_Transform.h"
+
+namespace Component
+{
+	class com_Physics
+	{
+	public:
+		com_Physics() {};
+		com_Physics( 
+			float mass, float invmass, float res,
+			glm::vec3 com, glm::mat4 inertia,
+			glm::vec3 vel = glm::vec3(0, 0, 0),
+			glm::vec3 acc = glm::vec3(0, 0, 0),
+			glm::vec3 rotvel = glm::vec3(0, 0, 0),
+			glm::vec3 rotacc = glm::vec3(0, 0, 0));
+
+		com_Physics(const com_Physics& p);
```

Comment:
until we need it, I think we shouldn't use the copy constructor. I'm making an effort to remove the other ones in components